CameraOverlay/tests/orchestrator_test.py

```diff
@@ -1,34 +1,40 @@
-"""Test Script For Orchestrator Class"""
+""" Test Script For Orchestrator Class """
 import orchestrator
 
 class TestOrchestrator:
-    """Orchestrator test class"""
+    """ Orchestrator test class """
     @staticmethod
     def test_class_instantiation():
-        """Test out class instantiation"""
-        test_broker_ip = 'localhost'
-        test_port = 1883
-        test_orchestrator = orchestrator.Orchestrator(test_broker_ip, test_port)
-        #Ensure broker ip and port are assigned
-        assert test_orchestrator.broker_ip == test_broker_ip, "failed - broker ip not assigned"
-        assert test_orchestrator.port == test_port, "failed - port not assigned"
+        """ Test out class instantiation """
+        broker_ip = 'localhost'
+        port = 1883
+        test_orchestrator = orchestrator.Orchestrator(broker_ip, port)
 
-        #Ensure MQTT client has not started
-        assert test_orchestrator.mqtt_client is None, "failed - MQTT client has started"
+         #Ensure broker ip and port are assigned
```

Comment:
Sorry I should have been more clear - the space goes after the `#`, before the actual comment